woocommerce/wc-admin
# Add/chart dashboard block

Fixes #690 

This PR introduces a `chart-block` to the dashboard, which wraps the `report-chart` component. The PR introduces a new `mode='block'` to the `report-chart` props to ensure that the legend is always fixed below the chart.

![block](https://user-images.githubusercontent.com/1160732/49937224-c1a7e200-fede-11e8-9037-8e9ba96ddd18.gif)


### Detailed test instructions:

- Open WooCommerce > Dashboard
- Ensure that the legend is on the bottom
- Ensure that there are no chart options in the chart header (line/bar type or interval selector)
- Hover over the chart to show the tooltip
